library/core/src/fmt/num.rs

```diff
@@ -598,12 +688,20 @@ impl u128 {
         issue = "none"
     )]
     pub fn _fmt<'a>(self, buf: &'a mut [MaybeUninit<u8>]) -> &'a str {
```

Comment:
Same here, this needs to be unsafe because it assumes the slice size. Or `_fmt_inner` should be made safe.